IssueComment
  { issueCommentUpdatedAt = 2014 (-06) (-25) 22 : 53 : 55 UTC
  , issueCommentUser =
      SimpleUser
        { simpleUserId = Id 1777134
        , simpleUserLogin = N "mlucy"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/1777134?v=3"
        , simpleUserUrl = "https://api.github.com/users/mlucy"
        , simpleUserType = OwnerUser
        }
  , issueCommentUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/comments/47169375"
  , issueCommentHtmlUrl =
      "https://github.com/rethinkdb/rethinkdb/issues/2589#issuecomment-47169375"
  , issueCommentCreatedAt = 2014 (-06) (-25) 22 : 53 : 55 UTC
  , issueCommentBody =
      "I ran the JS tests 255 times overnight, and while there were several errors (see #2606), this didn't re-occur.\r\n\r\nClosing since I can't reproduce and have no plausibly hypothesis about how it could happen.  We should re-open if it happens again."
  , issueCommentId = 47169375
  }